integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -148,7 +148,9 @@ case class CarbonLoadDataCommand(
             carbonLoadModel,
             factPath,
             dataFrame.isDefined,
-            optionsFinal)
+            optionsFinal,
+            options,
```

Comment:
Why passing to options, can you merge them into one?

Reply:
from finalOptions we will get all the default values of that particular option, but if we need only the options given from user,we need options also, so this one is added